webpack/ForemanInventoryUpload/Components/PageHeader/__tests__/PageHeader.test.js

```diff
@@ -1,13 +1,107 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { render, screen } from '@testing-library/react';
 import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
 
 import PageHeader from '../PageHeader';
 
+// Mock the useAdvisorEngineConfig hook
+jest.mock('../../../../common/Hooks/ConfigHooks', () => ({
+  useAdvisorEngineConfig: jest.fn(),
+}));
+
+import { useAdvisorEngineConfig } from '../../../../common/Hooks/ConfigHooks';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
```

Comment:
We have a helper for this in the latest Foreman: https://github.com/theforeman/foreman/blob/5205ccb014c196e67c822c41d67991925e53fa67/webpack/assets/javascripts/react_app/common/rtlTestHelpers.js#L18, but if we want to cherry-pick it to 3.16, I suppose we should not use it.